### ISSUE TITLE
Check systems failing the processCheck job

### ISSUE THREAD

**Issue by sxa** [AdoptOpenJDK/openjdk-infrastructure#1838]
We have the [SXA-processCheck job](https://ci.adoptopenjdk.net/view/Tooling/job/SXA-processCheck/) which checks for processes left over (usually from tests) on the machines. We should look at the non-green logs and see whether there are any tests that have specific recurring problems that we can resolve the underlying issues that are leaving processes around.

**Comment by Willsparker** [comment]
The two failing machines:  `test-ibmcloud-win2012r2-x64-*` , I have cleaned up as part of #1834 and #1824 . (Though I double checked them and they still only have the Jenkins Agent running)

Onto warning machines

**Comment by Willsparker** [comment]
I ran the [process check](https://ci.adoptopenjdk.net/view/Tooling/job/SXA-processCheck/133/) again quickly, just to clear out a lot of warnings, as I found an awful lot of the machines that used to report warning, have already been cleared out since it last run on the 11th. I'll go through and double check the ones that are still warning - however, I expect there's some that I won't have access to.

**Comment by Willsparker** [comment]
Machines I don't have access to that are reporting leftover processes:
```
testc-packet-fedora33-intel-1
```
The following machines reported warning with the most recent check that I linked above, but they were cleared up as part of the Jenkins Job:
```
build-scaleway-ubuntu1604-armv7-1  --> Process leftover in the Grinder folder
test-packet-ubuntu1604-armv8-1     --> Process leftover in the Test_openjdk11_j9_sanity.openjdk_aarch64_linux_xl folder
```
It appears a handful of the `test-aws-rhel76-armv8-` machines are offline (I assume as part of https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1843), so skipping those.

The only other warning was for `test-aws-ubuntu1804-armv8-1` - however that Jenkins run was from before Christmas, due to the machine being currently disabled. I manually checked, and there's no current leftover processes.

The job is looking much greener now, and I don't think the remaining warnings are necessarily any cause for concern.
